fitsnap3lib/io/outputs/pace.py

```diff
@@ -167,3 +169,40 @@ def _to_coeff_string(coeffs, config):
         out += "\n"
     out += "\n# End of potential"
     return out
+
+def _to_potential_file(config):
+    """
+    Use config settings to write a LAMMPS potential .mod file.
+    """
+
+    ps = config.sections["REFERENCE"].lmp_pairdecl[0]
+    ace_filename = config.sections["OUTFILE"].potential_name.split("/")[-1]
+
+    out = "# This file was generated by FitSNAP.\n"
+    out += f"# Hash: {config.hash}\n\n"
+
+    if "hybrid" in ps:
+        # extract non zero parts of pair style
+        if "zero" in ps.split():
+            split = ps.split()
+            zero_indx = split.index("zero")
+            del split[zero_indx]
+            del split[zero_indx] # delete the zero pair cutoff
+            ps = ' '.join(split)
+        out += ps + " pace product\n"
+        # add pair coeff commands from input, ignore if pair zero
+        for pc in config.sections["REFERENCE"].lmp_pairdecl[1:]:
+            out += f"{pc}\n" if "zero" not in pc else ""
+        pc_ace = f"pair_coeff * * pace {ace_filename}.yace"
+        for t in config.sections["ACE"].types:
+            pc_ace += f" {t}"
+        out += pc_ace
+    else:
+        out += "pair_style pace product\n"
```

Comment:
@jmgoff Should we be using `pace product` or `pace recursive` pair style? I recall recursive being faster?

Reply:
`pace product` should be fine